src/main/python/rerank/scripts/export_robust04_dataset.py

```diff
@@ -0,0 +1,52 @@
+'''scripts to generate the Robust04 dataset from the Anserini runs for reranking purpose
+
+Run BM25/BM25+RM3 on TREC Robust04 and store the output for reranking.
+Makes an external call to trec_eval for the actual computation of metrics.
+'''
+
+
+import json
+import os
+from tqdm import tqdm
+import string
+import argparse
+
+from utils import *
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--method', default='BM25', help='[BM25, BM25+RM3]')
+parser.add_argument('--output_dir', default='src/main/python/rerank/data', help='output directory')
+parser.add_argument('--K', default=1000, type=int, help='number of document retrieved')
+parser.add_argument('--index', default="/tuna1/indexes/lucene-index.robust04.pos+docvectors+rawdocs", help='index of Robust04 corpus')
+args = parser.parse_args()
+
+searcher = JSearcher(JString(args.index))
+
+fqrel = "src/main/resources/topics-and-qrels/qrels.robust2004.txt"
+qid2reldocids = get_qid2reldocids(fqrel)
+best_rm3_parameters = [[47, 9, 0.3], [47, 9, 0.3], [47, 9, 0.3], [47, 9, 0.3], [26, 8, 0.3]]
```

Comment:
Add a comment here about these being the parameters for five-fold from SIGIR Forum 2018 paper.

Reply:
Okay.